content/docs/deployment/csmoperator/_index.md

```diff
@@ -0,0 +1,253 @@
+
+
+## Installing CSI Driver via Operator
+
+[PowerScale Driver](/csmoperator/drivers/powerscale.md)
+
+**Note**: If you are using an OLM based installation, the example manifests are available in the `OperatorHub` UI.
+You can edit these manifests and install the driver using the `OperatorHub` UI.
+
+### Verifying the installation
+Once the driver Custom Resource has been created, you can verify the installation
+
+*  Check if ContainerStorageModule CR got created successfully
+
+    For e.g.
+    ```
+    $ kubectl get csm -n <driver-namespace>
+    ```
+* Check the status of the Custom Resource to verify if the driver installation was successful
+
+If the driver-namespace was set to _test-powerscale_, and the name of the driver is _powerscale_, then run the command `kubectl get csipowerscale/powerscale -n test-powerscale -o yaml` to get the details of the Custom Resource.
+
+Note: If the _state_ of the `CustomResource` is _Running_ then all the driver pods have been successfully installed. If the _state_ is _SuccessFul_, then it means the driver deployment was successful but some driver pods may not be in a _Running_ state.
+Please refer to the _Troubleshooting_ section [here](../../troubleshooting/operator) if you encounter any issues during installation.
+
+## Update CSI Drivers
+The CSI Drivers and CSM Modules installed by the Dell CSM Operator can be updated like any Kubernetes resource. This can be achieved in various ways which include –
+
+* Modifying the installation directly via `kubectl edit`
+    For e.g. - If the name of the installed PowerScale driver is powerscale, then run
+    ```
+    # Replace driver-namespace with the namespace where the PowerScale driver is installed
+    $ kubectl edit csm/powerscale -n <driver-namespace>
+    ```
+    and modify the installation
+* Modify the API object in-place via `kubectl patch`
+
+
+**NOTE:** `Replicas` in the driver CR file should not be greater than or equal to the number of worker nodes when upgrading the driver. If the `Replicas` count is not less than the worker node count, some of the driver controller pods would land in a pending state, and upgrade will not be successful. Driver controller pods go in a pending state because they have anti-affinity to each other and cannot be scheduled on nodes where there is a driver controller pod already running. Refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity for more details.
+### Supported modifications
+* Changing environment variable values for driver
+* Adding (supported) environment variables
+* Updating the image of the driver
+
+## Limitations
+* The Dell CSM Operator can't manage any existing driver installed using Helm charts. If you already have installed one of the Dell CSI driver in your cluster and  want to use the operator based deployment, uninstall the driver and then redeploy the driver following the installation procedure described above
```

Comment:
I think we should have a section at the very top that states it is applicable in a greenfield env, no existing driver/operator installed etc. Adding in here is a bit difficult for a reader to gather that info.

Reply:
Added greenfield environment at the top of the page in the "Note" section.